C#PlayersGuide.csproj

```diff
@@ -6,6 +6,16 @@
     <RootNamespace>C_PlayersGuide</RootNamespace>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <StartupObject></StartupObject>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
You can probably safely remove these artifacts of a previous build. Same with the `StartupObject` above. Try removing and building and see if it works.